src/sentry/static/sentry/app/components/createAlertButton.tsx

```diff
@@ -391,3 +406,13 @@ const StyledCloseButton = styled(Button)`
     opacity: 1;
   }
 `;
+
+const StyledLink = styled(Link)`
+  color: ${p => p.theme.bodyBackground};
+  text-decoration: underline;
```

Comment:
Let's do one off this, if we're gonna add links to tooltips, let's add these styles in the `Tooltip.tsx` itself. There's not really any reason this one tooltip with a link should be the only one to look like this.